libs/ui/select/brain/src/lib/brn-select-content.component.ts

```diff
@@ -0,0 +1,110 @@
+import { CdkListbox, ListboxValueChangeEvent } from '@angular/cdk/listbox';
+import { Component, ContentChild, ElementRef, HostListener, OnInit, inject, signal } from '@angular/core';
+import { takeUntilDestroyed, toObservable } from '@angular/core/rxjs-interop';
+import { HlmIconComponent } from '@spartan-ng/ui-icon-helm';
+import { tap } from 'rxjs';
+import { BrnSelectScrollDownDirective } from './brn-select-scroll-down.directive';
+import { BrnSelectScrollUpDirective } from './brn-select-scroll-up.directive';
+import { BrnSelectService } from './brn-select.service';
+
+// TODO: need to do logic for up and down arrows
+@Component({
+	selector: 'brn-select-content, hlm-select-content:not(noHlm)',
+	standalone: true,
+	imports: [BrnSelectScrollUpDirective, BrnSelectScrollDownDirective, HlmIconComponent],
```

Comment:
Agreed! Maybe we can work with content projection here?

Reply:
That is the same problem, that I had with the switch.
When using content projection, the api is not as nice as it would be now. :/
It is already possible to use content projection, hlm Icon is used if no brnSelectScrollUp or brnSelectScrollDown is provided in the content.
So technically this is not a "tight" dependency, it would also work without Hlm.

Reply:
Hmmm I had originally imagined that there would be a default icon and you would only need to provide the up/down icon arrows through projection if you wished to change the default to some other icon or icon family.  I see two potential options:

1. A brnIcon of some sort (would this just be ngIcon lol) - probably would not be different from current hlm really and would of course make ng-icon a dep for this component so probably not the best idea but an idea none the less.

2. We would have to require the user to project both and up and down icon and maybe throw some warning if they don't (provide both that is) and maybe just keep a placeholder text until hlmIcon contents are provided and or simply just keep this as an optional thing. If you don't provide content we can just leave this "hover scrolling areas" out.

Let me know what you guys think or open to any other suggestions also

Reply:
For me 2. sounds like the best of these options. 
The other option would be to make an helm component that uses the correct content, but this has the disadvantage of adding a new layer again. :/

Reply:
Yea I think lets just keep it as optional projections for these "hover scroll areas" optional. I'll make that change and i think just add some console warning if only one is provided not the other.